schemas/lioncore.serialization.schema.json

```diff
@@ -0,0 +1,520 @@
+{
```

Comment:
Did we decide to have two different serialization schemas?
Should this be stored in the organization repository if it must the same for all implementations?

Reply:
This one should also go to the `organization` repository, indeed.